### ISSUE TITLE
RNN-EM

### ISSUE THREAD

**Issue by OliviaZhi** [sjoerdvansteenkiste/Neural-EM#2]
Hi, in the paper, you mentioned "RNN-EM naturally extends to sequential data", does it mean that RNN-EM works better (than N-EM) in the "flying" experiments? Why the result shows that RNN-EM also works better than N-EM in the static case?

Thank you.

**Comment by sjoerdvansteenkiste** [comment]
Yes. In fact there is no principled way to directly apply N-EM to sequential data as the state across time-steps would not be carried over. In stead, one would need replace the mixture model with an HMM and derive the updates from there in the same way.

In the paper we have not opted for this, and therefore have only worked with RNN-EM on sequential data (merging EM iterations with time-steps and making use of the fact that it naturally carries a state across steps).
